cmd/jobbuzz-scraper/main.go

```diff
@@ -26,21 +25,25 @@ func main() {
 
 	service := service.Service{DB: db}
 
+	// Scrape JobCenter
+
 	fmt.Println("Fetching jobs from JobCenter")
 	jobs := scraper.ScrapeJobcenter()
 
 	for _, job := range jobs {
 		service.CreateJob(&job)
 	}
 
-	json_jobs, err := json.Marshal(jobs)
+	// _, err := json.Marshal(jobs)
 
-	if err != nil {
-		fmt.Println("Error json marshal", err)
-	}
+	// if err != nil {
+	// 	fmt.Println("Error json marshal", err)
+	// }
 
-	fmt.Println("Printing jobs from JobCenter")
-	fmt.Println(string(json_jobs))
+	// fmt.Println("Printing jobs from JobCenter")
+	// fmt.Println(string(json_jobs))
+
+	// Scrape Bruneida
 
 	fmt.Println("Fetching jobs from Bruneida")
 	jobs = scraper.ScrapeBruneida()
```

Comment:
if something went wrong here, should the program not stop or retry?

With how it is currently you are very likely to have some links fail and some pass and it would just be ignored and moved to the next step to be inserted into the database.

Reply:
I thought the scraping error has been handled using this "http://go-colly.org/docs/examples/error_handling/"?